start.sh

```diff
@@ -158,17 +158,24 @@ fi
 
 # Set the custom registry repo, defaulted to quay.io/open-cluster-management, but accomodate custom config focused on quay.io/acm-d for donwstream tests
 CUSTOM_REGISTRY_REPO=${CUSTOM_REGISTRY_REPO:-"quay.io/open-cluster-management"}
+# Default COMPOSITE_BUNDLE to true
+COMPOSITE_BUNDLE=${COMPOSITE_BUNDLE:-"true"}
 
 # If the user sets the COMPOSITE_BUNDLE flag to "true", then set to the `acm` variants of variables, otherwise the multicluster-hub version.  
 if [[ "$COMPOSITE_BUNDLE" == "true" ]]; then OPERATOR_DIRECTORY="acm-operator"; else OPERATOR_DIRECTORY="multicluster-hub-operator"; fi;
 if [[ "$COMPOSITE_BUNDLE" == "true" ]]; then CUSTOM_REGISTRY_IMAGE="acm-custom-registry"; else CUSTOM_REGISTRY_IMAGE="multicluster-hub-custom-registry"; fi;
 
+# Set the subscription channel, defaulted to snapshot-2.0
+if [[ "$COMPOSITE_BUNDLE" == "true" ]]; then SUBSCRIPTION_CHANNEL="release-2.0"; else CUSTOM_REGISTRY_IMAGE="snapshot-2.0"; fi;
```

Comment:
else branch should be setting SUBSCRIPTION_CHANNEL, not CUSTOM_REGISTRY_IMAGE, no?